components/library/libdnet/patches/dnet.3.patch

```diff
@@ -1,1489 +0,0 @@
---- libdnet-1.12/man/dnet.3.orig	2012-05-23 08:31:15.785689693 -0700
```

Comment:
Why did you remove these patches?

Reply:
Because they showed as already applied when I did gmake prep.

Reply:
No I misremembered, the guy actually changed the content of the tarball without changing the version so the patches do not apply anymore.

Reply:
Re-added.